### DISCUSSION TITLE
Instant filtering with search-as-you-type

### DISCUSSION THREAD

**Discussion by azzamsa** [sxyazi/yazi#247] [1. Q&A] [answered]
Hi.

Can we get "Instant filtering with search-as-you-type" akin to [nnn](https://github.com/jarun/nnn#features) ?

Thank you for yazi 🏅

**Comment by sxyazi** [comment, marked as answer]
Hi, it has been noted in the [Feature requests](https://github.com/sxyazi/yazi/issues/51), and if everything goes as planned, it will be shipped with Yazi v1.0.6. Of course, PRs are welcome to expedite this process!

**Comment by SZN3610** [comment]
😎

**Comment by SZN3610** [comment]
👻
